newm-server/src/main/kotlin/io/newm/server/aws/AwsKoinModule.kt

```diff
@@ -17,25 +18,26 @@ val awsKoinModule = module {
     single<AmazonS3> {
         AmazonS3ClientBuilder.standard()
             .withRegion(get<Regions>())
-            .withCredentials(get())
             .build()
     }
 
     single<AmazonSQSAsync> {
         AmazonSQSAsyncClientBuilder.standard()
             .withRegion(get<Regions>())
-            .withCredentials(get())
             .build()
     }
 
     single<AWSKMSAsync> {
         AWSKMSAsyncClientBuilder.standard()
             .withRegion(get<Regions>())
-            .withCredentials(get())
             .build()
     }
 
-    single<AWSCredentialsProvider> { CredentialsProvider(get()) }
+    single<AWSSecretsManagerAsync> {
```

Comment:
CredentialsProvider is gone. We should be able to get our credentials just from running on beanstalk.

Reply:
What about dev environment?